extras/EditorSupport/vim/syntax/sli.vim

```diff
@@ -0,0 +1,73 @@
+hi def link sliKeyword     Keyword
+
+syn keyword sliConstant M_INFO M_ERROR M_DEBUG M_WARNING M_STATUS M_FATAL
+hi def link sliConstant    Constant
+
+syn match sliUnit '\smV'
+syn match sliUnit '\spF'
+syn match sliUnit '\spA'
+syn match sliUnit '\sms'
+syn match sliUnit '\ss'
```

Comment:
In some cases this line causes a wrong highlighting. For example, the in command `M_ERROR setverbosity` the `s` is highlighted as unit. You could add a `space` after the declaration of the units to prevent that to happen.

After you change that, :+1: from me. Thanks for this work, I already used it several times!